doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst

```diff
@@ -107,8 +107,8 @@ Gauss quadrature points of the immersed mesh to represent the immersed body. For
       end
     end
 
-The ``beta`` coefficient is a parameter used to enforce the Nitsche IB. It's value is generally between 1 and 100, according to the size of the mesh. 
-A value of 10 is reasonable. Then, we specify the ``number of solids`` geometries that with be represented with Nitsche IB. 
+The ``beta`` coefficient is a parameter used to enforce the Nitsche IB. It's value is generally between 1 and 100 (10 being a reasonable value). The restriction automatically scales with the mesh size.
```

Comment:
I think you could refer to Jeanne's paper here.

Reply:
Well the paper does not really bring anything to the discussion here, so I would rather be succinct and just provide a value.